### PR TITLE
Design Calculator

### PR DESCRIPTION
**Pull request by akbarsazish**
1- Delete all the boilerplate from Create React App (text, images, styles).
2- Create a new directory called components.
3- Inside components, create a new Calculator.js file.
4- In Calculator.js, create a React component that matches this design.
5- Develop just the UI of the calculator
6- Use functional components

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+const ResultArea = () => <div type="text" className="resultArea"> 0 </div>;
+const CalcButtons = () => (
+  <div className="calcContainer">
+    <button type="button" className="calcItem btn-number">AC</button>
+    <button type="button" className="calcItem btn-number">+/-</button>
+    <button type="button" className="calcItem btn-number">%</button>
+    <button type="button" className="calcItem btn-sign">/</button>
```
Comment (julie-ify) [inline src/components/Calculator.js]: - [ ] Divide sign should be the same as the one we have in the design: "&div;". Kindly use that. If do not have it on your keyboard, then I would suggest the use of HTML entity code.

File: public/robots.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
+Disallow:
```
Comment (julie-ify) [inline public/robots.txt]: - [ ] Kindly remove this CRA file.

File: src/App.test.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```
Comment (julie-ify) [inline src/App.test.js]: - [ ] Kindly remove this generate boilerplate file.

File: src/reportWebVitals.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+      getCLS(onPerfEntry);
+      getFID(onPerfEntry);
+      getFCP(onPerfEntry);
+      getLCP(onPerfEntry);
+      getTTFB(onPerfEntry);
+    });
+  }
+};
+
+export default reportWebVitals;
```
Comment (julie-ify) [inline src/reportWebVitals.js]: - [ ] Kindly remove this generate boilerplate file.

File: src/setupTests.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
```
Comment (julie-ify) [inline src/setupTests.js]: - [ ] Kindly remove this generate boilerplate file.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @akbarsazish,

Good job so far!

## Highlights!
- Good code organization and design ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    <button type="button" className="calcItem btn-sign">-</button>
+    <button type="button" className="calcItem btn-number">1</button>
+    <button type="button" className="calcItem btn-number">2</button>
+    <button type="button" className="calcItem btn-number">3</button>
+    <button type="button" className="calcItem btn-sign">+</button>
+    <button type="button" className="calcItem btn-number btn-zero">0</button>
+    <button type="button" className="calcItem btn-number">.</button>
+    <button type="button" className="calcItem btn-sign"> = </button>
+  </div>
+);
```
Comment (julie-ify) [inline src/components/Calculator.js]: - [ ] **[OPTIONAL]** Kindly move these components to new files. Components are usually placed in their files in React.

**Comment by akbarsazish** [comment]
Hi @julie-ify 
Thank you so much for taking the time to review my code. I really appreciate your feedback, which was both helpful and constructive. I am sure my code is much better as a result.

**Review by Gambit142 — APPROVED** [review]
Hi @akbarsazish 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- Created a Calculator component ✔️ 
- Created a child component ✔️

### Optional Suggestions

- [ ] Kindly delete the following boilerplates:
- `public/favicon.ico`
- `public/manifest.json` 

#### Those boilerplates were created with create react app command and it is in the project requirement that all boilerplates created with that command should be deleted in this milestone. 👍 😎 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**